src/components/Reports/StaffExpenseReport/Helpers/formatDate.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { DateTime } from 'luxon';
+import { DateRange } from './StaffReportEnum';
+import {
+  dateRangeToString,
+  formatDate,
+  getFormattedDateString,
+} from './formatDate';
+
+/* The default now date is not helpful for these particular tests
+ * (for example, Year to Date would return January 1, 2020 - January 1, 2020),
+ * so it is mocked to a different date.
+ */
+jest
```

Comment:
This is a really helpful comment! Can you adjust this to mock the current time via `Settings.now = () => new Date(2020, 5, 18).valueOf()`?